python/mxnet/lr_scheduler.py

```diff
@@ -74,3 +74,52 @@ def __call__(self, num_update):
             logging.info("Update[%d]: Change learning rate to %0.5e",
                          num_update, self.base_lr)
         return self.base_lr
+
+class MultiFactorScheduler(LRScheduler):
```

Comment:
Consider merge with FactorScheduler and choose different behavior depending on step being a list or a number.

I don't have a very strong opinion on this so leave it as is if you think it's better.